MutexPractice/src/PrimeNo.java

```diff
@@ -0,0 +1,31 @@
+import com.sun.org.apache.xpath.internal.SourceTree;
+
+/**
+ * Created by Vishal on 7/15/2017.
+ */
+public class PrimeNo
+{
+    public static void main(String[] args)
+    {
+        int i = 0;
```

Comment:
Should not have variable name like 'i' unless and until you are using it i for loop